### ISSUE TITLE
Not able to install crouton

### ISSUE THREAD

**Issue by PinapplePizzaaa** [dnschneid/crouton#4515]
```
name: xenial
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/xenial...
crouton: version 1-20200724165050~master:08dac970
release: xenial
architecture: amd64
targets: unity,extension
host: version 13904.41.0 (Official Build) stable-channel hatch 
kernel: Linux localhost 4.19.185-29737-g391d4c3e561f #1 SMP PREEMPT Tue May 25 22:43:56 PDT 2021 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
Unmounting /mnt/stateful_partition/crouton/chroots/xenial...```

#### Please describe your issue:
towards the end of the installation, I get this:
```
Compiling CRAS (i386)...
In file included from libcras/cras_client.c:47:0:
libcras/cras_client.h:1439:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1454:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1475:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1491:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1506:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1530:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1550:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1570:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1592:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1608:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1623:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1638:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1653:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1668:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1683:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1732:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1753:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1767:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1784:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1799:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1815:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1831:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1847:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1878:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1893:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1908:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1924:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1940:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1955:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1970:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:1985:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
libcras/cras_client.h:2000:1: error: 'no_sanitize' attribute directive ignored [-Werror=attributes]
 {
 ^
cc1: error: unrecognized command line option '-Wno-int-in-bool-context' [-Werror]
cc1: all warnings being treated as errors
Failed to complete chroot setup.
Unmounting /mnt/stateful_partition/crouton/chroots/xenial...
```
any idea why this is? I ran sudo crouton -u afterwards and it still didn't do anything.

#### If known, describe the steps to reproduce the issue:
I just ran:
`sudo crouton -t unity,extension`

**Comment by caev** [comment]
This almost looks like a dup of #4450.

It also affects 'crouton -u' updates.

If I understand the dup bug correctly it can be fixed in [CRAS](https://github.com/dnschneid/crouton/issues/4450#issuecomment-835561942) or fixed in [Chrome](https://github.com/dnschneid/crouton/issues/4450#issuecomment-826182681).  Chrome is already fixed but not pushed to stable channel yet: I'm at '13904.55.0 (Official Build) stable-channel eve' but the Chrome fix is in 13912.0.0.

**Comment by DennisLfromGA** [comment]
@caev & @PinapplePizzaaa,

Yes, this is a dup of #4450 

Closing this, please follow up there, thanx.

-**DennisLfromGA**
